src/main/resources/application.yml

```diff
@@ -26,7 +26,7 @@ microservice:
   map:
     address: ${MAP_SERVICE}
 topics:
-  robot-movement: movement
+  robot-movement: movem
```

Comment:
Why did you change this? There are no changes to this topic's name in the specs.

Reply:
This was not intended